Create ant.yml

VishikBhalla opened the pull request:
https://github.com/junit-team/junit4/wiki/Getting-started-%E2%80%93-Ant 
for testing with Ant

VishikBhalla committed:
Create ant.yml

https://github.com/junit-team/junit4/wiki/Getting-started-%E2%80%93-Ant 
for testing with Ant

jackblanc commented:
Click details below this in the checks section, there's a syntax error in the .yml

VishikBhalla committed:
Update ant.yml

Possible fix. It said there was an error with line 21 even though there was no code on line 21 so I just deleted the previous line as it was not necessary.

VishikBhalla committed:
Create build.xml
